openlibrary/plugins/openlibrary/js/modals/index.js

```diff
@@ -2,8 +2,24 @@ import { FadingToast } from '../Toast.js';
 import '../../../../../static/css/components/metadata-form.less';
 
 
+/**
+ * Initializes share modal.
+ */
+
 export function initShareModal($modalLinks) {
     addClickListeners($modalLinks, '400px');
+    addShareModalButtonListeners();
+}
+/**
+ * Adds click listeners to buttons in all notes modals on a page.
+ */
+function addShareModalButtonListeners (){
+    $('#social-modal-content .copy-url-btn').on('click', function(event){
+        event.preventDefault();
+        navigator.clipboard.writeText(window.location.href);
+        showToast('URL copied successfully')
```

Comment:
@jimchamp question, is there a way we handle i18n in JS like this?

Reply:
Yes.  I'll include that in the follow-up issue (mentioned [here](https://github.com/internetarchive/openlibrary/pull/8806#issuecomment-1971633853)).